### ISSUE TITLE
[VMAccess] Does not work in RHEL-8

### ISSUE THREAD

**Issue by yuxisun1217** [Azure/azure-linux-extensions#800]
The VMAccess extension doesn't work in RHEL-8 on-demand image. 
There's no python or python3 in RHEL-8 on-demand image by default. It has "/usr/libexec/platform-python".  So that when running "./extension_shim.sh -i" it shows "No Python interpreter found on the box" and exit directly.

**Comment by johanburati** [comment]
It's an issue with the image not the extension, they need to set up python.

If you run the follow commands, **VMAccess** will works:
```
sudo yum install python36 -y
sudo ln /usr/bin/python3 /usr/bin/python
```

**Comment by johanburati** [comment]
@yuxisun1217  fyi I am trying to reach the ppl who built the image so that they fix this.

**Comment by paulmey** [comment]
There is python3, there's just no default `python` command

**Comment by johanburati** [comment]
yes, **waagent** uses **/usr/libexec/platform-python** from the **platform-python** package, all the python extensions use the unversioned python command which is not installed that why they are failing.

```
# waagent --version
WALinuxAgent-2.2.32.2 running on redhat 8.0
Python: 3.6.8
Goal state agent: 2.2.40
..
# /usr/bin/env python --version
/usr/bin/env: ‘python’: No such file or directory
```
Regarding the difference between the **platform-python** package and the **python36*** package, see the description:
```
Name         : platform-python
Version      : 3.6.8
Summary      : Internal interpreter of the Python programming language
Description  : This is the internal interpreter of the Python language for the system.
             : To use Python yourself, please install one of the available Python 3 packages,
             : for example python36.
```

I guess one could make point python to  platform-python, it is a bit messed up thought, also some extension does not work with **python3**
